dvc/commands/experiments/remove.py

```diff
@@ -3,24 +3,44 @@
 
 from dvc.cli.command import CmdBase
 from dvc.cli.utils import append_doc_link
+from dvc.exceptions import InvalidArgumentError
 
 logger = logging.getLogger(__name__)
 
 
 class CmdExperimentsRemove(CmdBase):
+    def raise_error_if_all_disabled(self):
```

Comment:
We raise an exception here if no explicit argument was provided. Same in `dvc exp pull/push`